applications/DigitalTwinApplication/python_scripts/processes/sensor_output_process.py

```diff
@@ -0,0 +1,79 @@
+from pathlib import Path
+import typing
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.DigitalTwinApplication as KratosDT
+
+from KratosMultiphysics.DigitalTwinApplication.utilities.sensor_utils import GetSensors
+from KratosMultiphysics.DigitalTwinApplication.utilities.sensor_utils import PrintSensorListToCSV
+from KratosMultiphysics.OptimizationApplication.utilities.optimization_problem import OptimizationProblem
+from KratosMultiphysics.OptimizationApplication.utilities.component_data_view import ComponentDataView
+from KratosMultiphysics.OptimizationApplication.responses.response_function import ResponseFunction
+
+def Factory(arg1: Kratos.Parameters, arg2: Kratos.Model, optimization_problem: 'typing.Optional[OptimizationProblem]' = None):
+    if isinstance(arg1, Kratos.Parameters) and isinstance(arg2, Kratos.Model):
+        # this is to use SensorOutputProcess as a normal kratos process
+        return SensorOutputProcess(arg2, arg1["Parameters"], optimization_problem)
+    elif isinstance(arg1, Kratos.Model) and isinstance(arg2, Kratos.Parameters):
```

Comment:
That is because, the if a normal `StructuralAnalysis` tries to create this process (which is required to generate the damaged system's measurement values) then first arg is the parameters, and then the model. In the `OptimizationApplication`, we have first argument `Model` and the `Parameters` to be consistent.